### ISSUE TITLE
multi-test: Store more data in ContractData

### ISSUE THREAD

**Issue by ethanfrey** [CosmWasm/cosmwasm-plus#350]
When we instantiate a contract, we just store the code id, indexed by it's address.

In `x/wasm`, we store quite a bit more: https://github.com/CosmWasm/wasmd/blob/master/proto/cosmwasm/wasm/v1/types.proto#L68-L84

I think minimally we should add:

```rust
pub struct ContractData {
  pub code_id: u64,
  pub creator: Addr,
  pub admin: Option<Addr>,
  pub label: String,
}
```

Maybe add some `created` field with the block height set?

We do not need to store this in protobuf, but the info should be available to query in the tests. (And we should enforce some sanity checks there)

**Comment by ethanfrey** [comment]
Good point. I fixed the link to wasmd

**Comment by ethanfrey** [comment]
Closed by #360
